test/functional/ios/appium_tests.py

```diff
@@ -85,11 +84,9 @@ def test_hide_keyboard(self):
         self.assertFalse(el.is_displayed())
 
     def test_hide_keyboard_presskey_strategy(self):
-        el = self.driver.find_element_by_name('Uses of UITextField')
-        el.click()
+        self._move_to_textbox()
 
-        # get focus on text field, so keyboard comes up
-        el = self.driver.find_element_by_class_name('UIATextField')
+        el = self.driver.find_elements_by_ios_predicate('wdType == "XCUIElementTypeTextField"')[0]
```

Comment:
Is it same as `find_element/s_by_class('XCUIElementTypeTextField')`?
(Other same predicates are also)

Reply:
Thanks, I reverted to `_by_class_name`.
(I'm not familiar with `iOS` for now, so any feedbacks are welcome.)